Examples/ObjectiveC/ExtrusionsExample.m

```diff
@@ -40,10 +40,11 @@ - (void)mapView:(MGLMapView *)mapView didFinishLoadingStyle:(MGLStyle *)style {
     layer.sourceLayerIdentifier = @"building";
     
     // Filter out buildings that should not extrude.
-    layer.predicate = [NSPredicate predicateWithFormat:@"height >= 0"];
+    layer.predicate = [NSPredicate predicateWithFormat:@"extrude == %@ AND height >= 0", @"true"];
```

Comment:
Nit: Per https://github.com/mapbox/mapbox-gl-native/issues/9056#issuecomment-302822052, this can be written `extrude == 'true'`.